lib/screens/alerts/cache/filter.ex

```diff
@@ -78,12 +86,31 @@ defmodule Screens.Alerts.Cache.Filter do
   end
 
   defp build_matcher({:stops, values}, acc) when is_list(values) do
-    matchers_for_values(acc, :stop, values)
-  end
+    route_mod = route_mod()
```

Comment:
Since this is now always the same per-Mix-env, it can be defined at compile time:

```elixir
@route_mod Application.compile_env(:screens, :alerts_cache_filter_routes_mod, Screens.Routes.Route)
```

This has the major advantage of preserving all the compile-time protections we'd normally get when calling a function directly, and allows Dialyzer to still work.

Reply:
The issue with defining this at compile time is that the Mock module is not defined at compile time so we run into compile warnings when running tests.

```
$ mix test
Compiling 5 files (.ex)
warning: Screens.Routes.Route.Mock.serving_stop/1 is undefined (module Screens.Routes.Route.Mock is not available or is yet to be defined)
  lib/screens/alerts/cache/filter.ex:94: Screens.Alerts.Cache.Filter.build_matcher/2
```